datajoint/connection.py

```diff
@@ -53,14 +53,13 @@ class Connection:
     """
 
     def __init__(self, host, user, passwd, init_fun=None):
-        self.erm = ERM(self)
         if ':' in host:
             host, port = host.split(':')
             port = int(port)
         else:
             port = config['database.port']
         self.conn_info = dict(host=host, port=port, user=user, passwd=passwd)
-        self._conn = pymysql.connect(init_command=init_fun, **self.conn_info)
+        self._conn = connector.connect(**self.conn_info)
```

Comment:
oh, this was because I tried another connector instead of `pymysql` and
only `pymysql` has the `init_command` option.  I will undo this. I will
redesign this part if we switch to other connectors.

On Mon, Aug 17, 2015 at 11:32 AM, Fabian Sinz <notifications@github.com>
wrote:

> In datajoint/connection.py
> <https://github.com/datajoint/datajoint-python/pull/142#discussion_r37206980>
> :
>
> >          if ':' in host:
> >              host, port = host.split(':')
> >              port = int(port)
> >          else:
> >              port = config['database.port']
> >          self.conn_info = dict(host=host, port=port, user=user, passwd=passwd)
> > -        self._conn = pymysql.connect(init_command=init_fun, **self.conn_info)
> > +        self._conn = connector.connect(**self.conn_info)
>
> Why is init_fun ignored now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datajoint/datajoint-python/pull/142/files#r37206980>.
>